docs/common-errors.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Common Errors
+---
+**In this section**
+
+* TOC
+{:toc}
+
+## Keywords for Relevant Log Lines.
+Users can search for relevant logs by searching the logs `connector-plugin`, `connector-plugin-warning`, and `connector-plugin-error`. 
+Search for the relevant logs in `/Users/<username>/Documents/My Tableau Repository/Logs/log.txt` for your current instance. 
+[Tableau Log Viewer](https://github.com/tableau/tableau-log-viewer) is a good tool to view and filter relevant logs when developing your connector. 
+
+
+##  Plugin Script Faliure
+If you get 
+  - `Running the plugin script failed`
+  - `Running the plugin script component exceeded the timeout: %1ms`
+  Check that you've correctly implemented `connectionBuilder.js`, `connectionProperties.js` and `required attributes` are correct. Please follow [this](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins/postgres_jdbc) example for the Postgres JDBC connector as the starting point for all the files necessary to successfully build a connector. 
+
+
+## Connector Doesn't show in Connector List
```

Comment:
Nitpick, but I'd put this above the Plugin script Failure one, guessing more people will be looking for this.